modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -429,6 +431,8 @@ public void testWriteTableAssignmentsToMetastoreExceptionally() throws Exception
      * @throws Exception If failed.
      */
     @Test
+    // TODO: https://issues.apache.org/jira/browse/IGNITE-24397 - remove @WithSystemProperty.
```

Comment:
Should we propagate this TODO for all annotations below? Compromise is to add `remove @WithSystemProperty there and below.`

Reply:
I don't think we need to do it. It is enough to search for `COLOCATION_FLAG` when cleaning up code after switching to colocation, all places that need to be addressed will be revealed.